JS/Lesson35/onder/my-app/src/components/UserPage/index.jsx

```diff
@@ -1,3 +1,19 @@
-export const UserPage = () => {
-  return (<div>User page</div>)
-};
\ No newline at end of file
+import React from "react";
+import { Typography, Paper, Container } from "@mui/material";
+
+const UserPage = () => {
```

Comment:
Since you're currently using route defined directly in the App.jsx component, UserPage is never displayed. You need to add a missing route for it similar to this: `<Route path="/user-settings" element={<UserPage />} />` to whereever you define routes.